dev_guide/builds.adoc

```diff
@@ -940,12 +940,8 @@ You can also use xref:combinations[combinations] of the these configurations
 to meet your specific needs.
 ====
 
-Builds are run with the *builder* service account, which must have access to any
-source secrets used. Access is granted with the following command:
-
-----
-$ oc secrets link builder mysecret
-----
+All source secrets must be linked to the builder account and added to the build
+configuration using the following instructions:
```

Comment:
ok on further review of the old/new content, you can ignore all of my original comment :)